### ISSUE TITLE
Getting error when trying to push commits

### ISSUE THREAD

**Issue by devang-ids** [jh3y/vincent-van-git#10]
cause of the error:
`git push -u origin main`

error:
```
error: src refspec main does not match any
error: failed to push some refs to 'https:/...'
```

can be easily fixed by adding `git branch -M main` to the script

**Comment by jh3y** [comment]
Hey @devang-ids 👋 

Thanks for pointing this out. Is this because you don't have a `main` branch on that repository? This is something I have set as the default branch name for git/Github. Did you try the script with that line added and it worked?

I'll happily make the change or accept a PR for it 🚀

Jhey ʕ •ᴥ•ʔ

**Comment by CompeyDev** [comment]
Been facing the same issue. PR #11 patches this.

![image](https://user-images.githubusercontent.com/74418041/158328617-42945316-8188-48bc-a754-080ad043e5fb.png)

**Comment by jh3y** [comment]
Closed with #11 🚀
